Documentation/Configuration/ConfigurationFiles.rst

```diff
@@ -6,6 +6,13 @@
 Configuration files
 ===================
 
+.. note::
+
+    This page is about configuration for the general TYPO3 installation.  
+    There might be many other configuration directives related to technical elements
```

Comment:
What are "technical elements"? Don't know the term.

Reply:
in the next line I wrote it, "like RTE ..."

Reply:
"There might be many other configuration directives related to technical elements"
This sentence contains no information to me.

Reply:
Of course, but it's not the purpose of this page to list all existing things but to inform about 2 distinct files.

Reply:
Everything as a technical element, TYPO3 itself with its extensions for example.

Reply:
Firstly the line is not the whole sentence but the next line explains it.  
Secondly I never feel that my expression is the only possible one, so feel free to replace it by another one if you think some other words could describe it better.

Reply:
Just remove this note.
It gives no new information at all. Most users know already that it is about TYPO3 configuration. And it does not help at all to only mention other possibilities without giving links to them.

Reply:
Then provide a PR for removing it again.

Reply:
Waste of time, I mean involving many people in a PR, ignoring other people's opinions, never coming up with a useful solution and just complaining in the end.